pyscriptjs/tests/unit/utils.test.ts

```diff
@@ -34,49 +33,22 @@ describe("Utils", () => {
   })
 })
 
-describe("CalculateFetchPaths", () => {
-  it("should calculate paths when only from is provided", () => {
-    const fetch_cfg: FetchConfig[] = [{from: "http://a.com/data.csv" }];
-    const [paths, fetchPaths] = calculatePaths(fetch_cfg);
-    expect(paths).toStrictEqual(["./data.csv"]);
-    expect(fetchPaths).toStrictEqual(["http://a.com/data.csv"]);
+describe("JoinPaths", () => {
+  it("should remove trailing slashes from the beginning and the end", () => {
+    const paths: string[] = ['///abc/d/e///'];
+    const joinedPath = joinPaths(paths);
+    expect(joinedPath).toStrictEqual('abc/d/e');
```

Comment:
this looks wrong.
If I do `jointPaths("/tmp", "foo.txt")` I expect `/tmp/foo.txt`, not `tmp/foo.txt`

Reply:
Done, I only check for a leading slash on the first element though. Otherwise it will be very cumbersome.